tests/unittests/Foundation/NSProgressTests.mm

```diff
@@ -134,26 +134,41 @@ - (void)addParentTo:(NSProgress*)child {
     ASSERT_EQ(YES, [progress isIndeterminate]);
 
     [progress setTotalUnitCount:5];
-    ASSERT_EQ(YES, [progress isIndeterminate]);
+    ASSERT_EQ(NO, [progress isIndeterminate]);
+
+    [progress setCompletedUnitCount:5];
+    ASSERT_EQ(NO, [progress isIndeterminate]);
+
+    [progress setTotalUnitCount:0];
+    ASSERT_EQ(NO, [progress isIndeterminate]);
 
     [progress setCompletedUnitCount:5];
+    [progress setTotalUnitCount:1];
     ASSERT_EQ(NO, [progress isIndeterminate]);
 
+    [progress setCompletedUnitCount:15];
+    [progress setTotalUnitCount:0];
+    ASSERT_EQ(NO, [progress isIndeterminate]);
+
+    [progress setCompletedUnitCount:0];
     [progress setTotalUnitCount:0];
     ASSERT_EQ(YES, [progress isIndeterminate]);
 }
 
-TEST(NSProgress, ChainImplicit) {
+OSX_DISABLED_TEST(NSProgress, ChainImplicit) {
     NSProgress* prog1 = [NSProgress progressWithTotalUnitCount:100];
     [prog1 becomeCurrentWithPendingUnitCount:100];
     NSProgress* prog2 = [NSProgress progressWithTotalUnitCount:50];
     [prog2 becomeCurrentWithPendingUnitCount:50];
     NSProgress* prog3 = [NSProgress progressWithTotalUnitCount:10];
     [prog3 setCompletedUnitCount:6];
 
+	
+	//We are getting 0, in osx
```

Comment:
looks like my auto formatter didn't do what it's supposed to do :(

---
In reply to: [73765478](https://github.com/Microsoft/WinObjC/pull/836#discussion_r73765478) [](ancestors = 73765478)